src/views/MainView.vue

```diff
@@ -1,30 +1,34 @@
 <template>
-  <div>
+  <div class="main-view">
     <div class="title text-center text-h4 font-weight-bold green--text text--darken-3">{{titleForMoviePage}}</div>
-    <MoviesList :movies="popularMovies" />
+    <MoviesList :movies="nowPlayingMovies" />
+    <AppPagination v-if="nowPlayingMovies" actionType="getNowPlayingMovies" />
   </div>
 </template>
 
 <script>
 import MoviesList from '@/components/MoviesList.vue'
+import AppPagination from '@/components/AppPagination.vue'
 import { TITLE } from '@/constants/titleConstants'
 
 export default {
-  name: 'NowPlayingView',
+  name: 'MainView',
   components: {
-    MoviesList
+    MoviesList,
+    AppPagination
   },
   created () {
     this.$store.dispatch('getNowPlayingMovies')
+    this.$store.dispatch('getConfigurationsForImages')
```

Comment:
Not in the MainView, in the App component, because user can start application from any page. Try to reload DetailsForMovie page

Reply:
I removed this method from application

Reply:
![1651084301541](https://user-images.githubusercontent.com/8670953/165600642-186e67cd-bc77-4092-bbb0-9ee3be216518.jpg)
You should also delete action call here because of error

Reply:
Sorry, I forgot(